ql/src/test/queries/clientpositive/udf5.q

```diff
@@ -9,24 +9,24 @@ SELECT from_unixtime(1226446340), to_date(from_unixtime(1226446340)), day('2008-
 SELECT from_unixtime(1226446340), to_date(from_unixtime(1226446340)), day('2008-11-01'), month('2008-11-01'), year('2008-11-01'), day('2008-11-01 15:32:20'), month('2008-11-01 15:32:20'), year('2008-11-01 15:32:20') FROM dest1_n14;
 
 EXPLAIN
-SELECT from_unixtime(unix_timestamp('2010-01-13 11:57:40', 'yyyy-MM-dd HH:mm:ss'), 'MM/dd/yy HH:mm:ss'), from_unixtime(unix_timestamp('2010-01-13 11:57:40')) from dest1_n14;
+SELECT from_unixtime(unix_timestamp('2010-01-13 11:57:40', 'uuuu-MM-dd HH:mm:ss'), 'MM/dd/yy HH:mm:ss'), from_unixtime(unix_timestamp('2010-01-13 11:57:40')) from dest1_n14;
```

Comment:
Instead of changing existing tests, can we add new ones with "uuuu"?

Reply:
Sure.